### PR TITLE
WE: Support the display of a context-menu when a user right clicks on a job row

### PR DESCRIPTION
**Pull request by AlexFalessi**
Pass Job data to the context menu scope before it is displayed

### CODE REVIEW

File: app/scripts/main.js [hunk 1]
```diff
@@ -355,12 +355,42 @@ mainModule.controller('mainController', function ($window, $http, $scope, $rootS
...
+                }
+            });
+            observer.observe($('#workflow-execution-main')[0], {
+                childList: true,
+                subtree: true
+            })
+        }
+    }
+
+    function closeWEJobRowActionsButtonDropDown() {
```
Comment (amouhoub) [inline app/scripts/main.js:387]: This function can be generic to all dropdown with the class .custom-dropdown.
I suggest that you rename this function to closeCustomDropdowns()

**Review by amouhoub — APPROVED** [review]
@AlexFalessi 
just a minor comment, plz apply it

**Review by mboussaa — APPROVED** [review]
Globally LGTM

**Comment by AlexFalessi** [comment]
retest this please

**Pull request merged**